packages/hub/cli.ts

```diff
@@ -17,9 +26,27 @@ yargs(hideBin(process.argv))
       console.log(yargs.help());
     }
     if (err) {
+      console.log('\n💡 While running:');
+      console.log(yargs.help());
```

Comment:
I can't find a good way to get information about the failing command in a `yargs.fail` handler, so using `yargs.help` for that, it is a bit verbose but knowing the failing command seems more important. Choosing not to log arguments in case a command ever requires secrets.